pkg/controller/virtualmachineimage/scratch_pvc.go

```diff
@@ -57,10 +57,12 @@ func newScratchPvc(vmi *hc.VirtualMachineImage, scheme *runtime.Scheme) (*corev1
 			Name:      getScratchPvcNameFromVmiName(vmi.Name),
 			Namespace: vmi.Namespace,
 		},
-		Spec: vmi.Spec.PVC,
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes:      []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+			Resources:        vmi.Spec.PVC.Resources,
+			StorageClassName: vmi.Spec.PVC.StorageClassName,
+		},
 	}
-	volumeMode := corev1.PersistentVolumeFilesystem
```

Comment:
why deleted this line? scratch pvc must be created with file system mode

Reply:
If `volumeMode` is not set, the default is file system mode.

Reply:
Even if the version of kubernetes upgrades, can we always guarantee that the default is filesystem in the future?

Reply:
We cannot guarantee it. So it need to be set to file.

Reply:
Done. @woohhan